x-pack/plugins/apm/common/alert_types.ts

```diff
@@ -5,6 +5,8 @@
  */
 
 import { i18n } from '@kbn/i18n';
+import { ValuesType } from 'utility-types';
+import { ANOMALY_SEVERITY, ANOMALY_THRESHOLD } from '../../ml/common';
```

Comment:
We are already exporting these from the root of our plugin.
```suggestion
import { ANOMALY_SEVERITY, ANOMALY_THRESHOLD } from '../../ml/public';
```